pallets/system/src/header_builder.rs

```diff
@@ -126,22 +124,39 @@ pub fn build_extension<M: Metrics>(
 	seed: Seed,
 	metrics: &M,
 ) -> HeaderExtension {
+	use once_cell::sync::Lazy;
+	static PMP: Lazy<kate::pmp::m1_blst::M1NoPrecomp> =
+		once_cell::sync::Lazy::new(|| kate::testnet::multiproof_params(256, 256));
```

Comment:
QQ: Does it work if block is 256 cols and 512 rows?
Keep in mind that rows & cols can be updated dynamically.

Reply:
Dynamically adjusting this would be nice. We should probably have a max size configged so this can init statically. Ok to do that in a separate PR?